eclass/xorg-3.eclass

```diff
@@ -273,19 +273,7 @@ xorg-3_src_unpack() {
 xorg-3_reconf_source() {
 	debug-print-function ${FUNCNAME} "$@"
 
-	case ${CHOST} in
-		*-aix* | *-winnt*)
-			# some hosts need full eautoreconf
-			[[ -e "./configure.ac" || -e "./configure.in" ]] \
-				&& XORG_EAUTORECONF=yes
-			;;
-		*)
-			# elibtoolize required for BSD
-			[[ ${XORG_EAUTORECONF} != no && ( -e "./configure.ac" || -e "./configure.in" ) ]] \
-				&& XORG_EAUTORECONF=yes
-			;;
-	esac
-
+	[[ ${XORG_EAUTORECONF} != no && ( -e "./configure.ac" || -e "./configure.in" ) ]] && XORG_EAUTORECONF=yes
 	[[ ${XORG_EAUTORECONF} != no ]] && eautoreconf
```

Comment:
unify these two lines into one common block?

Reply:
I don't think this even made sense before..

The problem is, the current/previous logic is:
* if `XORG_EAUTORECONF` is explicitly no, just elibtoolize.
* if `XORG_EAUTORECONF` is yes/unset, automagic it based on configure.{ac,in} presence
* if `XORG_EAUTORECONF` is unset, autoreconf

I don't see the point in the automagic, so let me clean it up.

Reply:
...it's dangerous in fact. How can you eautoreconf without ascertaining you actually have the deps?